README.md

```diff
@@ -84,6 +92,17 @@ Rounding can be used with maxPrecision.
 
 = `1.1K`
 
-### How it works
+## How it works
 
 If you're curious, have a look at the [CLDR data](https://github.com/unicode-cldr/cldr-numbers-modern) used under the hood. The logic we apply to the JSON is informed by documentation ([here](http://www.unicode.org/reports/tr35/tr35-29.html#Number_Format_Patterns) and [here](http://unicode.org/reports/tr35/tr35-numbers.html)) from Unicode. We use [Intl.js](https://github.com/andyearnshaw/Intl.js), which also relies on CLDR data, to inform our currency codes and number patterns.
+
+## Development instructions
+
+1. Install the specified Node version: `nvm install`
+1. Install dependencies: `npx yarn install`
```

Comment:
Why prefix all the `yarn` commands with `npx`? I think it is fair to assume that a consumer should either have yarn installed or know to use `npx` on an individual basis as needed.

Reply:
Hi Andy! 👋 These instructions were really "for Shopify" -- we don't have any sort of `dev.yml` so we don't actually have a global `yarn` installation we can use. For non-Shopifolk, they should use `yarn` directly like normal people.

We want to (eventually) add some better docs and upgrade our dependencies, so we will definitely have better instructions in the future.